FreeTubeApp/FreeTube
# Add the ability to view Invidious playlists

# Add the ability to view Invidious playlists

## Pull Request Type

## Description
This PR allows us to view Invidious playlists (playlists that were created on Invidious)

## Testing 
- enter this playlist in the search bar: https://vid.puffyan.us/playlist?list=IVPLl_MgDeoZc0LTKRPutqlmivaY8btU171
- see videos in playlist (even if using local api or different invidious instance is set)
- share playlist (see only IV options) 
![image](https://github.com/FreeTubeApp/FreeTube/assets/78101139/30286c72-9f27-4df5-98ec-4b9eb7bb1116)
- play a video from playlist
- wtach-video-playlist fills with videos from the IV playlist

## Desktop
- **OS:** Windows
- **OS Version:** 10
- **FreeTube version:** 0.19.1
